scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -4860,7 +4870,7 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
   }
 
   def templateStatSeq(enumCaseAllowed: Boolean = false): (Self, List[Stat]) = {
-    val emptySelf = autoPos(Self(autoPos(Name.Anonymous()), None))
+    val emptySelf = emptyPos(Self(emptyPos(Name.Anonymous()), None))
```

Comment:
Self is the only one that can be empty at a start of Indentation as far as I can tell.